umh.docs.umh.app/content/en/docs/features/datainfrastructure/unified-namespace.md

```diff
@@ -58,7 +60,8 @@ If you send the messages into other topics, some features might not work correct
   - `/` characters will be replaced with a `.`
   - and vice versa
 - By default, there will be no Authorization and Authentication on the MQTT broker. [You need to enable authentication and authorization yourself](/docs/production-guide/security/).
-- The MQTT or Kafka broker is not exposed externally by default. [You need to enable external MQTT access first](/docs/production-guide/administration/access-mqtt-outside-cluster/), or alternatively [expose Kafka externally](/docs/production-guide/administration/access-kafka-outside-cluster/).
+- The MQTT or Kafka broker is not exposed externally by default. [You need to enable external MQTT access first](/docs/production-guide/administration/access-services-from-outside-cluster/), or alternatively [expose Kafka externally](/docs/production-guide/administration/access-kafka-outside-cluster/).
+
 
 ## Where to get more information?
```

Comment:
We should update here the information about the data model. Also we should mention the use of the merge point, and what can be achieved with it. 

This requires first writing a user journey for the merge point feature. What is needed is not the exact user interaction, but instead all the other information. Who is using it, what is the business impact, etc.

Here as summary: 
- The merge point is used to increase the performance of Kafka. Additionally, it allows having more than a couple of hundred topics.

Some sources I could find: https://umh.docs.umh.app/docs/architecture/microservices/core/data-bridge/